pkgs/top-level/python-packages.nix

```diff
@@ -16790,4 +16790,24 @@ let
       maintainers = with maintainers; [ kamilchm ];
     };
   };
+
+  pafy = buildPythonPackage rec {
+    name = "pafy-${version}";
+    version = "0.3.74";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pafy/pafy-0.3.74.tar.gz";
+      md5 = "fbf0e7f85914eaf35f87837232eec09c";
+    };
+
+    propagatedBuildInputs = with self; [ youtube-dl ];
+
+    meta = with stdenv.lib; {
+      description = "Python library to download YouTube content and retrieve metadata.";
```

Comment:
Change description to `A library to download YouTube content and retrieve metadata`
Please notice that there is no dot in the end.
Thank you!